latestCommit.json

```diff
@@ -0,0 +1 @@
+f7d2e924bdff99886feddd529c922b6b99285ec0
```

Comment:
This file no longer exists as we are now saving the latest commit to an environment variable during the gulp task instead.